browser/ui/side_panel/ai_chat/ai_chat_side_panel_utils.h

```diff
@@ -0,0 +1,21 @@
+// Copyright (c) 2023 The Brave Authors. All rights reserved.
+// This Source Code Form is subject to the terms of the Mozilla Public
+// License, v. 2.0. If a copy of the MPL was not distributed with this file,
+// You can obtain one at https://mozilla.org/MPL/2.0/.
+
+#ifndef BRAVE_BROWSER_UI_SIDE_PANEL_AI_CHAT_AI_CHAT_SIDE_PANEL_UTILS_H_
+#define BRAVE_BROWSER_UI_SIDE_PANEL_AI_CHAT_AI_CHAT_SIDE_PANEL_UTILS_H_
+
+#include "content/public/browser/web_contents.h"
+
+class Browser;
+
+namespace ai_chat {
+
+// A helper function to get a Browser object from a WebContents.
+// TODO(b/276490341): Move this function to browser_finder.
```

Comment:
This TODO doesn't make sense because I don't know what that ID is. Perhaps link to the chromium version of this instead.

Reply:
Done